### PR TITLE
init

### PR DESCRIPTION
**Pull request by ttossavainen**
https://github.com/FrendsPlatform/Frends.CSV2/issues/1

### CODE REVIEW

File: Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    /// </summary>
+    /// <example>fi-FI</example>
+    public string CultureInfo { get; set; } = "";
+
+    /// <summary>
+    /// If set true, CSV's fields are never put in quotes
+    /// </summary>
+    /// <example>false</example>
+    [DefaultValue("false")]
+    public bool NeverAddQuotesAroundValues { get; set; }
```
Comment (Svenskapojkarna) [inline Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs]: This property is not used, so you are missing implementation for this feature.

File: Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    /// <summary>
+    /// Input's NULL values will be replaced with this value.
+    /// </summary>
+    /// <example>foo</example>
+    [DisplayFormat(DataFormatString = "Text")]
+    public string ReplaceNullsWith { get; set; }
```
Comment (Svenskapojkarna) [inline Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs]: This property is not used, so you are missing implementation for this feature.

File: Frends.CSV.Parse/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Frends.CSV.Parse
+Frends Task to parse string CSV content to a object.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.CSV/actions/workflows/Create_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.CSV/actions)
```
Comment (Svenskapojkarna) [inline Frends.CSV.Parse/README.md]: Copy-paste error in workflow link.

File: Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs (hunk 1 above)
Comment (ttossavainen) [reply Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs]: Was using Csv.Create as a template and forgot to remove this one.

File: Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs (hunk 2 above)
Comment (ttossavainen) [reply Frends.CSV.Parse/Frends.CSV.Parse/Definitions/Options.cs]: Was using Csv.Create as a template and forgot to remove this one.

**Review by Svenskapojkarna — APPROVED** [review]

**Pull request merged**